pyvalidator/is_freight_container.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+def is_valid_container_number(container_number)-> bool:
```

Comment:
The input argument missing the type definition.

If the input is a string, utilise the `assert_string` function. Check the usage of it through other validators _(is_url as example)_.